utils/config/wallets.ts

```diff
@@ -1,24 +1,136 @@
-import { PolkadotjsWallet } from '~/utils/config/wallets/PolkadotjsWallet'
-import { MathWallet } from '~/utils/config/wallets/MathWallet'
-import { NovaWallet } from '~/utils/config/wallets/NovaWallet'
-import { SubWallet } from '~/utils/config/wallets/SubWallet'
-import { TalismanWallet } from '~/utils/config/wallets/TalismanWallet'
-import { EnkryptWallet } from '~/utils/config/wallets/EnkryptWallet'
 import { isMobileDevice } from '~/utils/extension'
 
-// source as 'polkadot-js' in mobile app
+// wallet logo
+import logoPolkadotjs from '@/assets/partners/logo-polkadot-js.png'
+import logoClover from '@/assets/partners/logo-clover.png'
+import logoLedger from '@/assets/partners/logo-ledger.svg'
+import logoMathWallet from '@/assets/partners/logo-mathwallet.png'
+import logoNova from '@/assets/partners/logo-nova.png'
+import logoSubWallet from '@/assets/partners/logo-subwallet.svg'
+import logoTalisman from '@/assets/partners/logo-talisman.svg'
+import logoEnkrypt from '@/assets/partners/logo-enkrypt.png'
+import { BaseDotsamaWallet } from './wallets/BaseDotsamaWallet'
+
 export enum SupportWalletExtension {
   PolkadotJs = 'polkadot-js',
   MetaMask = 'metamask',
   Clover = 'clover',
   Ledger = 'ledger',
-  Math = 'polkadot-js', // mathwallet
-  Nova = 'polkadot-js', // nova
+  Math = 'mathwallet',
+  Nova = 'nova',
```

Comment:
iirc this should stay as `polkadot-js` to be launched on mobile app

Reply:
The logic was moved here
<img width="767" alt="image" src="https://user-images.githubusercontent.com/31397967/198057473-1b242c90-7524-43d0-9a99-e5dbff22948e.png">

Reply:
### Enums should not have duplicate values
ref: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1069

Reply:
![522x0](https://user-images.githubusercontent.com/9987732/198058316-72327a6a-e3cc-4c01-a484-d596c5c7123b.jpg)